src/main/java/seedu/address/model/person/TeleHandle.java

```
@@ -0,0 +1,57 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.util.Objects;
+
+/**
+ * Represents a Person's Telegram handle in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidTeleHandle(String)}
+ */
+public class TeleHandle {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Telegram handles should be of the format @handle";
+
+    public static final String VALIDATION_REGEX = "@[\\p{Alnum}]*";
```

Currently, the following still works but I think it should throw an error.
`add n/Ben m/CS2100 e/ben123@gmail.com h/`
`add n/Ben m/CS2100 e/ben123@gmail.com h/@` 

Can we change it to: 
```
public static final String VALIDATION_REGEX = "@[\\p{Alnum}]*[^\\s]";
```

This should reject any whitespace telegram inputs.

---

Same suggestion as the ModuleCode